ivy/functional/ivy/manipulation.py

```diff
@@ -850,6 +850,42 @@ def stack(
     return res
 
 
+@handle_exceptions
+@handle_backend_invalid
+@handle_nestable
+@handle_array_like_without_promotion
+@handle_view
+@handle_out_argument
+@to_native_arrays_and_back
+@handle_array_function
+@handle_device_shifting
```

Comment:
This seems to be undefined 🤔

Reply:
honestly I don't have any idea what these decorators do XD I just copied them from the `reshape` function because I assumed that `unflatten` has the same behavior as `reshape`.

Reply:
You also have to import them. You can copy the import as well from where you picked this up 😄